x-pack/plugins/security_solution/server/lib/detection_engine/rules/get_rules_to_update.test.ts

```diff
@@ -4,102 +4,470 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { getRulesToUpdate } from './get_rules_to_update';
+import { filterInstalledRules, getRulesToUpdate, mergeExceptionLists } from './get_rules_to_update';
 import { getResult } from '../routes/__mocks__/request_responses';
 import { getAddPrepackagedRulesSchemaDecodedMock } from '../../../../common/detection_engine/schemas/request/add_prepackaged_rules_schema.mock';
 
 describe('get_rules_to_update', () => {
-  test('should return empty array if both rule sets are empty', () => {
-    const update = getRulesToUpdate([], []);
-    expect(update).toEqual([]);
-  });
+  describe('get_rules_to_update', () => {
+    test('should return empty array if both rule sets are empty', () => {
+      const update = getRulesToUpdate([], []);
+      expect(update).toEqual([]);
+    });
 
-  test('should return empty array if the id of the two rules do not match', () => {
-    const ruleFromFileSystem = getAddPrepackagedRulesSchemaDecodedMock();
-    ruleFromFileSystem.rule_id = 'rule-1';
-    ruleFromFileSystem.version = 2;
+    test('should return empty array if the id of the two rules do not match', () => {
```

Comment:
Super nit: in the test descriptions do you want to specify that it's the `ruleId` we're checking? I know it's intuitive looking at the tests and logically.

Reply:
Yep, changed this comment now. Thank you